lib/modules/manager/gradle-wrapper/artifacts.spec.ts

```diff
@@ -187,10 +187,52 @@ describe('modules/manager/gradle-wrapper/artifacts', () => {
     expect(result).toEqual([
       {
         artifactError: {
-          lockFile: 'gradle-wrapper.properties',
+          lockFile: 'gradle/wrapper/gradle-wrapper.properties',
           stderr: 'Response code 404 (Not Found)',
         },
       },
     ]);
   });
+
+  it('handles gradle-wrapper in subdirectory', async () => {
+    git.getRepoStatus.mockResolvedValue({
+      modified: [
+        'sub/gradle/wrapper/gradle-wrapper.properties',
+        'sub/gradlew',
+        'sub/gradlew.bat',
+      ],
+    } as StatusResult);
+
+    const execSnapshots = mockExecAll();
+
+    const res = await gradleWrapper.updateArtifacts({
+      packageFileName: 'sub/gradle/wrapper/gradle-wrapper.properties',
+      updatedDeps: [],
+      newPackageFileContent: await readString(
+        `./expectedFiles/gradle/wrapper/gradle-wrapper.properties`
+      ),
+      config: { ...config, newValue: '6.3' },
+    });
+
+    expect(res).toEqual(
+      [
+        'sub/gradle/wrapper/gradle-wrapper.properties',
+        'sub/gradlew',
+        'sub/gradlew.bat',
+      ].map((fileProjectPath) => ({
+        file: {
+          type: 'addition',
+          path: fileProjectPath,
+          contents: 'test',
+        },
+      }))
+    );
+    expect(execSnapshots).toMatchObject([
+      {
+        options: {
```

Comment:
```suggestion
        cmd: '<gradlew> wrapper --gradle-version 5.6.4',
        options: {
```

we need to make sure it's called with right path